### PR TITLE
Expand Cyborg

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
add: Cyborgs can now be upgraded to be h-u-g-e! Only a cosmetic effect!
/:cl:

[why]:
Since growth serum is in the game, I figured borgs could use something like this. This gives no advantage to the player and is purely cosmetic for the hell of it.

It's locked behind several layers of research, requires 2* the metal needed to construct a borg + 5 sheets of titanium.
Video shows old size:
https://www.youtube.com/watch?v=dL6h43yVi0I&feature=youtu.be`
New size so kevinz doesn't feel insecure:
dojxbsz 1

### CODE REVIEW

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -387,3 +387,35 @@
...
+	smoke.start()
+	sleep(2)
+	for(var/i in 1 to 4)
+		playsound(R, pick('sound/items/drill_use.ogg', 'sound/items/jaws_cut.ogg', 'sound/items/jaws_pry.ogg', 'sound/items/welder.ogg', 'sound/items/ratchet.ogg'), 80, 1, -1)
+		sleep(12)
+	if(!prev_lockcharge)
+		R.SetLockdown(0)
+	R.anchored = FALSE
+	R.notransform = FALSE
+	R.resize = 2.5
```
Comment (kevinz000) [inline code/game/objects/items/robot/robot_upgrades.dm]: 2.

Comment (kevinz000) [reply code/game/objects/items/robot/robot_upgrades.dm]: please. just 2. seriously.

Comment (DaxDupont) [reply code/game/objects/items/robot/robot_upgrades.dm]: 2 what

Comment (Buggy123) [reply code/game/objects/items/robot/robot_upgrades.dm]: 2.

**Comment by Buggy123** [comment]
E X P A N D C Y B O R G

File: code/game/objects/items/robot/robot_upgrades.dm (hunk 1 above)
Comment (DaxDupont) [reply code/game/objects/items/robot/robot_upgrades.dm]: Fine sec

**Comment by Jalleo** [comment]
That isnt a cyborg. THAT IS A TANK MY FRIEND, Best quality material made from burning stations.

**Comment by Buggy123** [comment]
Add a admin only version that turns the cyborg red and makes them into a god

**Comment by Xhuis** [comment]
`e x p a n d`

**Comment by AverageJoe82** [comment]
you are confused, this is already **the body of a god**

File: code/game/objects/items/robot/robot_upgrades.dm (hunk 1 above)
Comment (PKPenguin321) [reply code/game/objects/items/robot/robot_upgrades.dm]: s m a l l

**Comment by F-OS** [comment]
inflation fetish fuel

**Comment by Kromgar** [comment]
I don't understand the point of this

**Comment by ShadowDeath6** [comment]
tbh I'd like if it costed more and made the borg slightly tankier but a lot slower, I don't really see a point of a meme item that takes up a borg upgrade slot.

**Comment by DoctoreMad** [comment]
@Kromgar It doesn't have a point, no need to be autiste about it.

**Comment by DaxDupont** [comment]
It doesn't have a point like 80% of the game, it's just something silly and
neat to have.

**Comment by The-clown-known-as-Pepe** [comment]
code bloat

**Comment by DaxDupont** [comment]
@The-clown-known-as-Pepe yeah sure why not

**Comment by 81Denton** [comment]
>code bloat

Yeah no shit look at the size of that thing

**Comment by ExcessiveUseOfCobblestone** [comment]
This plus vtec if that still exists will be hilarious

**Comment by TimeIsEndless** [comment]
Finally a useful PR that adds something that wont break the game for the next 2 weeks

**Pull request merged**